src/tarantool-gdb.py

```diff
@@ -0,0 +1,468 @@
+        ( 'replicaset::on_ack', 'trigger::link' ),
+        ( 'replicaset::applier::on_rollback', 'trigger::link' ),
+        ( 'replicaset::applier::on_wal_write', 'trigger::link' ),
+        ( 'schema_module::funcs', 'func_c::item' ),
+        ( 'space::before_replace', 'trigger::link' ),
+        ( 'space::child_fk_constraint', 'fk_constraint::in_child_space' ),
+        ( 'space::ck_constraint', 'ck_constraint::link' ),
+        ( 'space::memtx_stories', 'memtx_story::in_space_stories' ),
+        ( 'space::on_replace', 'trigger::link' ),
+        ( 'space::parent_fk_constraint', 'fk_constraint::in_parent_space' ),
+        ( 'space::space_cache_pin_list', 'space_cache_holder::link' ),
+        ( 'swim::dissemination_queue', 'swim_member::in_dissemination_queue' ),
+        ( 'swim::on_member_event', 'trigger::link' ),
+        ( 'swim::round_queue', 'swim_member::in_round_queue' ),
+        ( 'swim_scheduler::queue_output', 'swim_task::in_queue_output' ),
+        ( 'tx_manager::read_view_txs', 'txn::in_read_view_txs' ),
+        ( 'tx_manager::all_stories', 'memtx_story::in_all_stories' ),
+        ( 'tx_manager::traverse_all_stories', 'memtx_story::in_all_stories' ), # struct rlist*
+        ( 'tx_manager::all_txs', 'txn::in_all_txs' ),
+        ( 'txn::conflict_list', 'tx_conflict_tracker::in_conflict_list' ),
+        ( 'txn::conflicted_by_list', 'tx_conflict_tracker::in_conflicted_by_list' ),
+        ( 'txn::gap_list', 'gap_item::in_gap_list' ),
+        ( 'txn::full_scan_list', 'full_scan_item::in_full_scan_list' ),
+        ( 'txn::on_commit', 'trigger::link' ),
+        ( 'txn::on_rollback', 'trigger::link' ),
+        ( 'txn::on_wal_write', 'trigger::link' ),
+        ( 'txn::point_holes_list', 'point_hole_item::in_point_holes_list' ),
+        ( 'txn::read_set', 'tx_read_tracker::in_read_set' ),
+        ( 'txn::savepoints', 'txn_savepoint::link' ),
+        ( 'txn_limbo::queue', 'txn_limbo_entry::in_queue' ),
+        ( 'txn_stmt::on_commit', 'trigger::link' ),
+        ( 'txn_stmt::on_rollback', 'trigger::link' ),
+        ( 'sql_stmt_cache::gc_queue', 'stmt_cache_entry::link' ),
+        ( 'sys_alloc::allocations', 'container::rlist' ),
+        ( 'user::credentials_list', 'credentials::in_user' ),
+        ( 'vy_cache_env::cache_lru', 'vy_cache_node::in_lru' ),
+        ( 'vy_history::stmts', 'vy_history_node::link' ),
+        ( 'vy_join_ctx::entries', 'vy_join_entry::in_ctx' ),
+        ( 'vy_lsm::on_destroy', 'trigger::link' ),
+        ( 'vy_lsm::runs', 'vy_run::in_lsm' ),
+        ( 'vy_lsm::sealed', 'vy_mem::in_sealed' ),
+        ( 'vy_lsm_recovery_info::ranges', 'vy_range_recovery_info::in_lsm' ),
+        ( 'vy_lsm_recovery_info::runs', 'vy_run_recovery_info::in_lsm' ),
+        ( 'vy_quota::wait_queue', 'vy_quota_wait_node::in_wait_queue' ),
+        ( 'vy_range::slices', 'vy_slice::::in_range' ),
+        ( 'vy_range_recovery_info::slices', 'vy_slice_recovery_info::in_range' ),
```

Comment:
just a little bit :)
I consider 80 symbols limit only as recommendation, not as a strict rule. Here we have a lot of items, one per line, so I assume it's not a big deal if a few of them cross 80 sym boundary. I think visually it's more consistent than it would be if some items occupy 2 line while others occupy 1.

I would keep current approach